client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -54,16 +55,25 @@ const NewStatsNotices = ( { siteId, isOdysseyStats }: StatsNoticesProps ) => {
 /**
  * Return new or old StatsNotices components based on env.
  */
-export default function StatsNotices( { siteId, isOdysseyStats }: StatsNoticesProps ) {
+export default function StatsNotices( {
+	siteId,
+	isOdysseyStats,
+	statsPurchaseSuccess,
+}: StatsNoticesProps ) {
 	const statsAdminVersion = useSelector( ( state: object ) =>
 		getJetpackStatsAdminVersion( state, siteId )
 	);
+
 	const supportNewStatsNotices =
 		! isOdysseyStats ||
 		!! ( statsAdminVersion && version_compare( statsAdminVersion, '0.10.0-alpha', '>=' ) );
 
 	return supportNewStatsNotices ? (
-		<NewStatsNotices siteId={ siteId } isOdysseyStats={ isOdysseyStats } />
+		<NewStatsNotices
+			siteId={ siteId }
+			isOdysseyStats={ isOdysseyStats }
+			statsPurchaseSuccess={ statsPurchaseSuccess }
+		/>
```

Comment:
Since post-purchase notices behave so differently compared to the "NewStatsNotices" (which use the new stats notices API), let's create a new component to contain the post-purchase notices:

```suggestion
		<>
			<NewStatsNotices
				siteId={ siteId }
				isOdysseyStats={ isOdysseyStats }
			/>
			<PostPurchaseNotices
				siteId={ siteId }
				statsPurchaseSuccess={ statsPurchaseSuccess }
			/>
		</>
```

Reply:
Fixed in f91e3426625cabbc165e9587f99c95e885142317.